packages/react/src/index.ts

```diff
@@ -123,6 +123,7 @@ export { UnspecifiedError } from '@lens-protocol/api-bindings';
 export { Amount, WellKnownSymbols, ether, matic, erc20, usd } from '@lens-protocol/shared-kernel';
 export * from './sources';
 export * from './utils';
+export { PromiseResultPresenter } from './transactions/adapters/PromiseResultPresenter';
```

Comment:
Let's make sure all `PromiseResultPresenter.ts` exports are flagged as `@internal`

Reply:
I ended up removing the export of PromiseResultPresenter and implementing a version directly in react-web